src/preferences/dialog/dlgprefsound.cpp

```diff
@@ -370,6 +412,22 @@ void DlgPrefSound::loadSettings(const SoundManagerConfig &config) {
         deviceSyncComboBox->setCurrentIndex(0);
     }
 
+    if (m_config.getAPI() == MIXXX_PORTAUDIO_JACK_STRING) {
+        // TODO(Be): Get the processing latency from JACK. PortAudio does
+        // not support this as of June 2017.
+        roundTripLatencySpinBox->setMinimum(0.0);
+    } else {
+        bool measuredRoundTripLatencyAtDefault =
+            m_pRoundTripLatency->get() == roundTripLatencySpinBox->minimum();
+        roundTripLatencySpinBox->setMinimum(config.getProcessingLatency() * 2.0);
```

Comment:
I am not going to test with JACK. I do not expect new results only by changing the sound API.
How about bringing this discussion to an end and just follow this 
https://github.com/mixxxdj/mixxx/pull/1279#issuecomment-312518730